index.html

```diff
@@ -760,8 +758,8 @@ <h2>
               <li data-tests=
               "payment-request-ctor-currency-code-checks.https.html">
                 <a>Check and canonicalize total amount</a>
-                |details|.<a>total</a>.<a data-lt=
-                "PaymentItem.amount">amount</a>. Rethrow any exceptions.
+                |details|.<a>total</a>.{{PaymentItem/amount}}. Rethrow any
```

Comment:
```suggestion
                |details|.{{PaymentDetailsInit /total}}.{{PaymentItem/amount}}. Rethrow any
```

Reply:
ok. removing the `data-lt-for` in this `<li>` as well since it is no longer needed once we do this. will do this locally so that i can remove `data-lt-for` and add this suggestion in the same commit.